pkg/controller/propagator/propagation.go

```diff
@@ -417,9 +424,14 @@ func processTemplates(replicatedPlc *policiesv1.Policy, decision appsv1.Placemen
 
 			if !isConfigurationPolicy(policyT) {
 				// has Templates but not a configuration policy
-				err = errors.NewBadRequest("Templates are restricted to only Configuration Policy")
+				err = errors.NewBadRequest("Templates are restricted to only Configuration Policies")
 				log.Error(err, "Not a Configuration Policy")
-				return err
+
+				r.recorder.Event(rootPlc, "Warning", "PolicyPropagation",
```

Comment:
Can you make it an error rather than a warning?

Reply:
@mprahl : I did look into this a bit ,   everywhere in GRC controllers only have a Normal or Warning -  even for policy violations  its a warning.

Online Search  also appears to bring up  only Normal or Warning as Event types.
.
Even a major problem like Pod crashing or not coming up due to ImagePull appears to be recorded as a Warning type.

I am unsure of repercussions of  setting an Error state, like its effects  to the overall reporting of health of the clusters. Have you ever used Error state before ?  

 Also given that we are going to set a violation on the policy status itself soon, which is what the user actually sees, I dont know that this has much value to it. ?